Tests/scripts/infrastructure_tests/test_collect_tests.py

```diff
@@ -124,11 +124,12 @@ class MockerCases:
 ALWAYS_INSTALLED_PACKS = ('Base', 'DeveloperTools')
 
 
-def _test(monkeypatch, case_mocker: CollectTestsMocker, collector_class: Callable,
+def _test(mocker, monkeypatch, case_mocker: CollectTestsMocker, collector_class: Callable,
           expected_tests: Iterable[str], expected_packs: Iterable[str], expected_packs_to_upload: Iterable[str],
           expected_machines: Iterable[Machine] | None,
           expected_modeling_rules_to_test: Iterable[str | Path] | None,
-          collector_class_args: tuple[Any, ...] = ()):
+          collector_class_args: tuple[Any, ...] = (),
+          disable_collect_packs_diff_master_bucket=True):
```

Comment:
add to docstrings, explain why we disable it only sometimes

Reply:
Done